.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+  "parser": "@babel/eslint-parser",
```

Comment:
This is not among dependencies, and this is a TypeScript project without a single js file so this is not needed to be configured here with ESLint. We only want the typescript eslint parser for everything